submissions/JuliaNovak/html-css-popup/styles.css

```diff
@@ -0,0 +1,189 @@
+:root {
+    --hover-color: rgb(103, 103, 200);
+    --bg-color:rgb(245, 243, 243);
+    --secondary-bg-color: lightgrey;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.menu {
+    display: flex;
+    justify-content: end;
+    align-items: center;
+    padding: 0 1rem;
+    background-color: lightblue;
+}
+
+.nav-items-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.menu li{
```

Comment:
Avoid styling by HTML tags.
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css